src/app/forms/templates/trl/trl-tyres.template.ts

```diff
@@ -0,0 +1,100 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeTypes } from '@forms/services/dynamic-form.types';
+
+export const tyresTemplateTrl: FormNode = {
+  name: 'tyreSection',
+  type: FormNodeTypes.GROUP,
+  label: 'Tyres',
+  children: [
+    {
+      name: 'tyreUseCode',
+      label: 'Tyre use code',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      validators: [
+        { name: ValidatorNames.MaxLength, args: 2 },
+        { name: ValidatorNames.Min, args: 0 }
+      ]
+    },
+    {
+      name: 'axles',
+      value: '',
+      type: FormNodeTypes.ARRAY,
+      children: [
+        {
+          name: '0',
+          label: 'Axle',
+          value: '',
+          type: FormNodeTypes.GROUP,
+          children: [
+            {
+              name: 'axleNumber',
+              label: 'Axle Number',
+              type: FormNodeTypes.CONTROL,
+              validators: [
+                { name: ValidatorNames.Numeric },
```

Comment:
If the axle number is editable and meant to be of type number can you use the NUMBER edit type please. if it's not editable are the validators necessary?